### PR TITLE
add GitHub and Npm links to Storybook header

### PR DESCRIPTION
**Pull request by alcail**
# What
added links before Search components
image

# Why
User Story https://skype.visualstudio.com/SPOOL/_workitems/edit/2653670

# How Tested
ran storybook locally

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Comment by anjulgarg** [comment]
THIS IS BRILLIANT !!!!
@alcail :o

File: packages/storybook/.storybook/manager-head.html [hunk 1]
```diff
@@ -78,4 +78,74 @@
...
+  function addUsefulLinks() {
+    const linkStyle = 'color: black; text-decoration: none; text-align: center; cursor: pointer; padding: 0rem 0.5rem;';
+    const imageStyle = 'height: 0.75rem; margin-top: 0.125rem;';
+    const textStyle = 'margin-left: 0.25rem; font-size: 0.75rem;';
+    const linkContentStyle = 'display: flex;';
+    
+    const sidebarHeader = document.getElementsByClassName('sidebar-header');
+    if (sidebarHeader.length === 0) {
+      // sidebar container has not loaded in the page yet, retry in 500ms
+      console.log('sidebar-header not loaded');
```
Comment (mgamis-msft) [inline packages/storybook/.storybook/manager-head.html:93]: Should we log this on console?

Comment (alcail) [reply packages/storybook/.storybook/manager-head.html:93]: nop, forgot to remove this one. good catch :)

**Review by mgamis-msft — APPROVED** [review]

**Comment by prprabhu-ms** [comment]
5w8muk

**Review by prprabhu-ms — APPROVED** [review]

**Pull request merged**